app/src/common/shared/com/igalia/wolvic/browser/Services.kt

```diff
@@ -84,7 +85,7 @@ class Services(val context: Context, places: Places): WSession.NavigationDelegat
             }
         }
     }
-    val serverConfig = ServerConfig(Server.RELEASE, CLIENT_ID, REDIRECT_URL)
+    val serverConfig = ServerConfig(if (BuildConfig.FXA_USE_CHINA_SERVER) Server.CHINA else Server.RELEASE, CLIENT_ID, REDIRECT_URL)
```

Comment:
I'm not sure this is very readable, embedding a conditional as a function argument. Also causes line to get very long.
Maybe resolve server as a variable beforehand?

Reply:
I guess I'm used to very long lines in WebKit or Chromium :smile: . If you don't mind we can clean it up in a follow up patch.